pkg/didcomm/protocol/didexchange/persistence/connection.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/storage"
 )
 
+// ConnectionData contain info about did exchange connection
+type ConnectionData struct {
```

Comment:
I might have expected `ConnectionRecord` with the other naming in this package like recorder.

p.s., I don't think the Data suffix is really needed (as it's in a persistence package).

Reply:
Rename to ConnectionRecord. Connection name is already used in did exchange pkg https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didcomm/protocol/didexchange/models.go#L72